src/CMakeLists.txt

```diff
@@ -14,6 +14,17 @@ target_include_directories(Agent PUBLIC ${CMAKE_SOURCE_DIR}/include)
 add_executable(wazuh-agent agent/src/main.cpp)
 target_link_libraries(wazuh-agent Agent Logger)
 
+if(WIN32)
+    set(MANIFEST_FILE "${CMAKE_SOURCE_DIR}/app.manifest")
```

Comment:
This could be moved into a `src/cmake/SetWindowsManifest.cmake` file (name could be different), and reduce this to a function call. But we can also do this later if the cmake file keeps growing.

Reply:
Done!